packages/core/addon/models/chart-visualization.ts

```diff
@@ -83,25 +68,27 @@ export default class ChartVisualization extends Visualization {
    * @returns series config object
    */
   buildDimensionSeries(
-    config: DimensionSeriesConfigSerialized,
-    validations: TODO,
+    config: string,
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    validations: { attrs: any },
```

Comment:
It is an object but since it's from ember-cp-validations and based on the deeply nested property strings I didn't think it would be particularly useful to try and type it